sir_complainsalot/delphi_sir_complainsalot/run.py

```diff
@@ -37,14 +37,6 @@ def run_module():
                                        params["sources"], params.get("grace", 0), LOGGER))
 
     if len(complaints) > 0:
-        for complaint in complaints:
-            LOGGER.critical(event="signal out of SLA",
-                            message=complaint.message,
-                            data_source=complaint.data_source,
-                            signal=complaint.signal,
-                            geo_types=complaint.geo_types,
-                            last_updated=complaint.last_updated.strftime("%Y-%m-%d"))
```

Comment:
This argument is dropped below.  Was that intentional?

Reply:
Yeah, in the new code it's been pulled directly into the aggregated message.